runtime/src/dpe_crypto.rs

```diff
@@ -45,8 +48,21 @@ impl<'a> Crypto for DpeCrypto<'a> {
     type Hasher<'b> = DpeHasher<'b> where Self: 'b;
     type PrivKey = KeyId;
 
-    fn rand_bytes(&mut self, _dst: &mut [u8]) -> Result<(), CryptoError> {
-        Err(CryptoError::NotImplemented)
+    fn rand_bytes(&mut self, dst: &mut [u8]) -> Result<(), CryptoError> {
+        let mut curr_idx = 0;
+        while curr_idx < dst.len() {
+            let trng_bytes = <[u8; 48]>::from(
+                self.trng
+                    .generate()
+                    .map_err(|_| CryptoError::CryptoLibError)?,
+            );
+            let bytes_to_write = min(dst.len() - curr_idx, 48);
+            dst.get_mut(curr_idx..curr_idx + bytes_to_write)
+                .ok_or(CryptoError::Size)?
+                .copy_from_slice(&trng_bytes[..bytes_to_write]);
```

Comment:
nit: I wonder if doing a normal subslice (`dst[curr_idx..curr_idx + bytes_to_write].copy_from_slice(&trng_bytes[..bytes_to_write]`) doesn't panic now. Not a big deal either way though. If it panics, what you have here is fine.